### ISSUE TITLE
Components menu pushed down

### ISSUE THREAD

**Issue by xjensen** [cagov/design-system#618]
On component pages, the mobile components menu is getting pushed down below the first heading.

<img width="543" alt="Screen Shot 2022-02-03 at 13 35 12" src="https://user-images.githubusercontent.com/1208960/152433322-aa314d78-8670-4cca-a6a2-bc80291b9dbb.png">

I had a fun morning trying to find the source of this bug. It's due to #571. 

I feel this should be done in the site's templates rather than the page-navigation component. Page-navigation (and content sites in general) should not have a monopoly on the sidebar. People will need to use the sidebar for other things. This is the kind of "Wait, what just happened?" code that's really going to trip people up when they try to use our Design System.

I can make this change but wanted to check and see if I'm missing anything. cc @aaronhans @kkoryaka

**Comment by aaronhans** [comment]
Yeah that makes sense, better not to have that logic inside the component. Thanks @xjensen

**Comment by kkoryaka** [comment]
Yes, make sense. So, should we close this #571 issue for these reasons?

**Comment by jaredwyee** [comment]
Confirmed that component menu is ABOVE now as expected. Pixel 3a chrome: 

![image](https://user-images.githubusercontent.com/85529382/153273562-59f3288c-a6ba-40dd-96ee-2b73871898af.png)
